_posts/2022-04-04-inclusive-language-guide.md

```diff
@@ -0,0 +1,25 @@
+---
+category: blog-detail
+post-type: blog
+by: the Scala Center
+title: "The Inclusive Language Guide for the Scala community"
+---
+
+As Scala stewards, we should all be committed to providing a friendly, safe and welcoming environment for all, regardless of age, body size, disability, ethnicity, sex characteristics, gender identity and expression, level of experience, education, socio-economic status, nationality, personal appearance, race, religion, sexual identity and orientation, or other such characteristics.
+
+The Scala Center aims to maintain and promote an environment that is representative of the rich diversity of our users and community.
+This is why, as requested by its advisory board, the Scala Center created an Inclusive Language Guide to help the community adopt a more inclusive way of communicating.
```

Comment:
"requested by its advisory board" could link to https://github.com/scalacenter/advisoryboard/blob/main/proposals/025-inclusive-language.md

partly to indirectly credit the proposer, partly just to raise awareness of the whole proposal process

Reply:
Good point! Thanks.